### PR TITLE
[OSDOCS#9122]: 4.13.27 release notes

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-9122

Link to docs preview:
https://69787--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-27

QE review:
N/A

Additional information:
*NOTE*: URLs will not work until release is live

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -3603,6 +3603,23 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.13.26 --pullspecs
 ----
 
-[id="ocp-4-13-26-updating"]
+[id="ocp-4-13-27-updating"]
 ==== Updating
```
Comment (cbippley) [inline release_notes/ocp-4-13-release-notes.adoc:3607]: The updating section goes last.

**Review by cbippley — COMMENTED** [review]
Make sure you move the updating section to the end (while keeping the one for 4.13.26 in place). Other than that, LGTM!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-13-release-notes.adoc:3607]: ahh :) done!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -3606,3 +3606,23 @@ $ oc adm release info 4.13.26 --pullspecs
 [id="ocp-4-13-26-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-27"]
+=== RHSA-2023:7837 - {product-title} 4.13.27 bug fix and security update
+
+Issued: 2024-01-04
+
+{product-title} release 4.13.27, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7837[RHSA-2023:7837] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:7829[RHBA-2023:7829] advisory.
```
Comment (JoeAldinger) [inline release_notes/ocp-4-13-release-notes.adoc:3615]:
```suggestion
{product-title} release 4.13.27, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7827[RHSA-2023:7827] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:7829[RHBA-2023:7829] advisory.
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -3606,3 +3606,23 @@ $ oc adm release info 4.13.26 --pullspecs
 [id="ocp-4-13-26-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-27"]
+=== RHSA-2023:7837 - {product-title} 4.13.27 bug fix and security update
```
Comment (JoeAldinger) [inline release_notes/ocp-4-13-release-notes.adoc:3611]:
```suggestion
=== RHSA-2023:7827 - {product-title} 4.13.27 bug fix and security update
```

**Review by JoeAldinger — COMMENTED** [review]
Just need to swap the number of the image advisory

**Pull request merged**